pkg/client_data/lib/package_api.dart

```diff
@@ -88,6 +88,24 @@ class Message {
   Map<String, dynamic> toJson() => _$MessageToJson(this);
 }
 
+@JsonSerializable()
+class PackageData {
+  final String name;
+  final VersionInfo latest;
+  final List<VersionInfo> versions;
```

Comment:
Can we add a note in `VersionInfo` to document that as of Dart 2.8 `pub` client uses `archive_url` to find the archive.

Also we can add a documentation comment that these are sorted by version number... how they are sorted.